src/test/unit/AllocationManagerUnit.t.sol

```diff
@@ -3,22 +3,35 @@ pragma solidity ^0.8.27;
 
 import "src/contracts/core/AllocationManager.sol";
 import "src/test/utils/EigenLayerUnitTestSetup.sol";
+import "src/test/mocks/MockAVSRegistrar.sol";
```

Comment:
We don't have tests for some view functions:

`getMaxMagnitudesAtBlock`
`getStrategyAllocations`

Reply:
@0xClandestine can you add these to your testing / TODO PR and get them in a followup?

I'd like to get this merged as it's a big change and we've been sitting on it for a long time.